views/widget.scout.display.php

```diff
@@ -48,7 +48,7 @@
 		if ( ! isset( $after_title ) ) {
 			$after_title = '';
 		}
-		echo $before_title . wp_kses_post( trim( $title ) ). $after_title;
+		echo esc_html( $before_title ) . wp_kses_post( trim( $title ) ). esc_html( $after_title);
```

Comment:
**nit-pick**: This file uses a combination of tabs and spaces. While you are here, can you clean it up to follow the Wordpress coding standard:

[PHP](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation
) and [HTML](https://make.wordpress.org/core/handbook/best-practices/coding-standards/html/#indentation) Indentation: 
> Rule of thumb: Tabs should be used at the beginning of the line for indentation, while spaces can be used mid-line for alignment.

Reply:
You can setup your editor to show whitespace (ex. [visual code](https://stackoverflow.com/questions/30140595/show-whitespace-characters-in-visual-studio-code))

Reply:
Changed it to this:
`echo esc_html( $before_title ) . wp_kses_post( trim( $title ) ) . esc_html( $after_title );`

Reply:
@henrivrod I'm not actually talking about your code. I'm talking about the file as a whole. Let me know if you'd like to chat on zoom - it might be easier to show you in my editor.